teamA/jinwoo/models/userDao.js

```diff
@@ -0,0 +1,54 @@
+const { DataSource } = require('typeorm');
+
+const mysqlDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE
+})
+
+mysqlDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!")
+  })
+  .catch((err) => {
+    console.error("Error during Data Source initialization", err);
+      mysqlDataSource.destroy();
+  });
+
+  const createUser = async ( name, email, password, profileImage ) => {
+    try {
+      return await mysqlDataSource.query(
+        `INSERT INTO usersWesta(
+          name,
+          email,
+          password,
+          profile_image
+        ) VALUES (?, ?, ?, ?);
+        `,
+        [ name, email, password, profileImage ]
+      );
+    } catch (err) {
+      const error = new Error('INVALID_DATA_INPUT');
+      error.statusCode = 500;
+      throw error;
+    }
+  };
+
+  const readPosts = async (userId) => {
```

Comment:
```suggestion
  const getPostsByUserId = async (userId) => {
```

해당 코드는 users보다 posts 관련으로 옮겨져야 보다 올바른 위치에 존재 할 수 있을 것 같습니다!
여러 계층의 다른 코드들도 적용시켜주세요~